src/server/exports/errorResponse.js

```diff
@@ -0,0 +1,5 @@
+const create = (message) => {
+    return {error: message};
+}
+
+exports.create = create;
```

Comment:
This object makes your code more readable. I mean in this case it's a tiny impact, but you got the message and can implement it later on in a bigger context.